axiom/orgs.go

```diff
@@ -21,7 +21,7 @@ type Plan uint8
 const (
 	emptyPlan Plan = iota //
 
-	Hobby      // hobby
+	Personal   // personal
 	Basic      // basic
 	Team       // team
```

Comment:
Will this handle the new `teamMonthly` plan?

Reply:
Fixed.